main.js

```diff
@@ -0,0 +1,64 @@
+let library = [];
+
+const getInput = () => {
+  const book = {};
+  const titleInput = document.getElementById('bookTitle');
+  const authorInput = document.getElementById('bookAuthor');
+
+  // Return null if either title or author input is empty
+  if (titleInput.value.trim() === '' || authorInput.value.trim() === '') {
+    return null;
+  }
+
+  book.title = titleInput.value;
+  book.author = authorInput.value;
+  return book;
+};
+
+const removeBook = (title) => {
+  const book = document.getElementById(title);
+  book.remove();
+  library = library.filter((bookObj) => bookObj.title !== title);
+  localStorage.setItem('library', JSON.stringify(library));
+};
+
+const addBook = (bookObj) => {
+  const bookList = document.getElementById('book-list');
+  const book = document.createElement('LI');
+  book.setAttribute('id', bookObj.title);
+  const titleElement = document.createElement('h3');
+  titleElement.style.margin = '0'; // Remove margin
+  titleElement.textContent = bookObj.title;
+  const authorElement = document.createElement('p');
+  authorElement.style.margin = '0'; // Remove margin
+  authorElement.textContent = bookObj.author;
+  const deleteBtn = document.createElement('button');
+  deleteBtn.innerHTML = 'Remove';
+  deleteBtn.addEventListener('click', () => removeBook(bookObj.title));
+  book.appendChild(titleElement);
+  book.appendChild(authorElement);
+  book.appendChild(deleteBtn);
+  bookList.appendChild(book);
```

Comment:
- [OPTIONAL] After adding a book, I noticed that the form inputs are not cleared. I would advise that you clear the form inputs after a user adds a book for a better user experience. 👍 💻